airflow/decorators/base.py

```diff
@@ -62,8 +62,8 @@
         OperatorExpandArgument,
         OperatorExpandKwargsArgument,
     )
-    from airflow.models.mappedoperator import ValidationSource
     from airflow.sdk.definitions.dag import DAG
+    from airflow.sdk.definitions.mappedoperator import ValidationSource
```

Comment:
unrelated: `ValidationSource` is a weird name but it existed before the PR anway